lib/cartopy/crs.py

```diff
@@ -3219,6 +3219,59 @@ def x_limits(self):
     def y_limits(self):
         return self._y_limits
 
+class Spilhaus(Projection):
+    """
+    Spihaus-Adams World Ocean Map in a Square II
```

Comment:
Small typo missing the L in Spilhaus. Where does the II come from? Is there a good reference for this, or link out to the PROJ documentation?

https://proj.org/en/stable/operations/projections/spilhaus.html

Reply:
Yes, it is essentially this map projection with a different pole https://proj.org/en/stable/operations/projections/adams_ws2.html. I think it will be clearer if I just remove the "Adams" and the "II".